3/test/excercise3.spec.js

```diff
@@ -0,0 +1,110 @@
+const { gql } = require('apollo-server')
+const server = require('../server.js')
+const {createTestClient} = require('apollo-server-testing');
+const jwt = require('jsonwebtoken');
+
+const { query, mutate } = createTestClient(server);
+
+describe('Query: Given a list of todos and assignees', () => {
+
+    const queryTodoLength = gql`query getCountTodos {countTodos}`;
+    const queryAssigneeLength = gql`query getCountAssignees {countAssignees}`;
+    
+    it('get all todo items at startup', () => {
+        query({query: queryTodoLength}).then((result) => {
+            expect(result.data.countTodos).toBe(6);
+        })
+    })
+    it('get all assignees at startup', () => {
+        query({query: queryAssigneeLength}).then((result) => {
+            expect(result.data.countAssignees).toBe(3);
+        })
+    })
+
+    const querySignedAssignee = gql`query getSigneAssignee($token: String!) { assignees(token: $token) {id, name} }`;   
+    const tokenMessage = jwt.sign({auth: 'der richtige inhalt'}, 'shhhhh');
+
+    it('get all assignees signed', () => {
+        query({query: querySignedAssignee, variables: {token: tokenMessage}}).then((result) => {
+            expect(result.data.assignees.length).toBe(3);
+        })
```

Comment:
I am not sure about this one, but I think that JWTs belong into the request header and should not be provided in the body of the request